specification/cognitiveservices/data-plane/Speech/BatchTextToSpeech/stable/2024-04-01/batchsynthesis.json

```diff
@@ -0,0 +1,779 @@
+        "id": {
+          "$ref": "#/definitions/ResourceId"
+        },
+        "description": {
+          "description": "The description of the batch synthesis task.",
+          "type": "string"
+        },
+        "createdDateTime": {
+          "description": "The time-stamp when the object was created.",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "BatchSynthesisBackgroundAudioDefinition": {
+      "description": "Background audio setting.",
+      "required": [
+        "src"
+      ],
+      "type": "object",
+      "properties": {
+        "src": {
+          "format": "uri",
+          "description": "Source of audio file. Reference https://learn.microsoft.com/en-us/azure/ai-services/speech-service/speech-synthesis-markup-voice#add-background-audio.",
+          "type": "string"
+        },
+        "fadein": {
+          "format": "int64",
+          "description": "The fadein value of background audio. Reference https://learn.microsoft.com/en-us/azure/ai-services/speech-service/speech-synthesis-markup-voice#add-background-audio.",
+          "type": "integer"
+        },
+        "fadeout": {
+          "format": "int64",
+          "description": "The fadeout value of background audio. Reference https://learn.microsoft.com/en-us/azure/ai-services/speech-service/speech-synthesis-markup-voice#add-background-audio.",
+          "type": "integer"
+        },
+        "volume": {
+          "format": "double",
+          "description": "The volume value of background audio. Reference https://learn.microsoft.com/en-us/azure/ai-services/speech-service/speech-synthesis-markup-voice#add-background-audio.",
+          "type": "number"
+        }
+      }
+    },
+    "BatchSynthesisBillingDetails": {
+      "description": "Bath synthesis billing details.",
+      "type": "object",
+      "properties": {
+        "neuralCharacters": {
+          "format": "int64",
+          "description": "The number of billable characters for prebuilt neural voices in the input file.",
+          "type": "integer"
+        },
+        "customNeuralCharacters": {
+          "format": "int64",
+          "description": "The number of billable characters for custom neural voices in the input file.",
+          "type": "integer"
+        },
+        "aoaiCharacters": {
+          "format": "int64",
+          "description": "The number of billable characters for AOAI voices in the input file.",
+          "type": "integer"
+        },
+        "aoaiHDCharacters": {
+          "format": "int64",
+          "description": "The number of billable characters for AOAI HD voices in the input file.",
+          "type": "integer"
+        },
+        "personalVoiceCharacters": {
+          "format": "int64",
+          "description": "The number of billable characters for personal voices in the input file.",
+          "type": "integer"
+        }
+      },
+      "readOnly": true
```

Comment:
The `readOnly` attribute should only be specified in schema property definitions.

Reply:
Agree. I think the `readOnly` should be on parent schema `billingDetails` property?

Similar for the `BatchSynthesisError`.

Reply:
Thank you and fixed.